compiler/closureiters.nim

```diff
@@ -155,6 +155,9 @@ type
     nearestFinally: int # Index of the nearest finally block. For try/except it
                     # is their finally. For finally it is parent finally. Otherwise -1
 
+const
+  nkAtoms = { nkEmpty..nkNilLit, nkTemplateDef, nkTypeSection } + procDefs
```

Comment:
'nkAtoms' would just be nkEmpty..nkNilLit, but yours is ``toSkip`` or ``nkSkip``